.lintstagedrc

```diff
@@ -1,6 +1,5 @@
 {
     "*.{js,md,ts,yml,json}": [
-        "prettier --write",
-        "git add"
```

Comment:
this was raising a warning:
```
Some of your tasks use `git add` command. Please remove it from the config since all modifications made by tasks will be automatically added to the git commit index.
```

Reply:
I think that "_prettier --write_" should remain here